packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -145,6 +147,20 @@ export type CallCompositeOptions = {
    * if this is not supplied, the composite will not show a 'network troubleshooting' link.
    */
   onNetworkingTroubleShootingClick?: () => void;
+  /* @conditional-compile-remove(unsupported-browser) */
+  /**
+   * Callback you may provide to supply users with a provided page to showcase supported browsers by ACS.
+   *
+   * @example
+   * ```ts
+   * onBrowserTroubleShootingClick?: () =>
+   *  window.open('https://contoso.com/browser-troubleshooting', '_blank');
+   * ```
+   *
+   * @remarks
+   * if this is not supplied, the composite will not show a unsupported browser page.
+   */
+  onBrowserTroubleShootingClick?: () => void;
```

Comment:
`TroubleShooting` -> `Troubleshooting`

Reply:
We should update onNetworkingTroubleShootingClick at some point as well (in a seperate PR)

Reply:
Can we update this to `onEnvironmentUnsupportedTroubleshootingClick`? Its a bit verbose but given we are checking the environmentInfo it would be nice to be consistent